packages/react-components/react-nav-preview/src/components/NavGroup/useNavGroup.styles.ts

```diff
@@ -0,0 +1,245 @@
+import type { NavGroupSlots, NavGroupState } from './NavGroup.types';
+
+import { makeStyles, mergeClasses, shorthands } from '@griffel/react';
+import { createCustomFocusIndicatorStyle } from '@fluentui/react-tabster';
+import { tokens, typographyStyles } from '@fluentui/react-theme';
+import { SlotClassNames } from '@fluentui/react-utilities';
+
+export const navGroupClassNames: SlotClassNames<NavGroupSlots> = {
+  root: 'fui-NavGroup',
+  icon: 'fui-NavGroup__icon',
+  content: 'fui-NavGroup__content',
+};
+
+const reservedSpaceClassNames = {
+  content: 'fui-NavGroup__content--reserved-space',
+};
+
+// These should match the constants defined in @fluentui/react-icons
+// This package avoids taking a dependency on the icons package for only the constants.
+const iconClassNames = {
+  filled: 'fui-Icon-filled',
+  regular: 'fui-Icon-regular',
+};
+
+/**
+ * Styles for the root slot
+ */
+/* eslint-disable @typescript-eslint/naming-convention */
+const useRootStyles = makeStyles({
```

Comment:
For base styles we typically use [`makeResetStyles`](https://griffel.js.org/react/api/make-reset-styles) to cut down on generated class names.

Reply:
Neat! TIL :-)